README.md

```diff
@@ -3,7 +3,9 @@ cloverage
 
 Simple clojure coverage tool.
 
-Travis: [![Build Status](https://secure.travis-ci.org/cloverage/cloverage.png?branch=master)](https://travis-ci.org/cloverage/cloverage), CircleCi: [![CircleCI](https://circleci.com/gh/cloverage/cloverage.svg?style=svg)](https://circleci.com/gh/cloverage/cloverage)
+Travis: [![Build Status](https://travis-ci.org/cloverage/cloverage.svg?branch=master)](https://travis-ci.org/cloverage/cloverage)
+
+CircleCi: [![CircleCI](https://circleci.com/gh/cloverage/cloverage.svg?style=svg)](https://circleci.com/gh/cloverage/cloverage)
```

Comment:
Mind if I do that in a separate PR? I did this with the web UI, so adding another change is annoying.